src/DISPUTE_MANAGEMENT.md

```diff
@@ -0,0 +1,19 @@
+# Dispute Management
+
+The purpose of this document is to provide guidelines to Product Teams in regards to how to settle disputes about situations that may arise when working with the Auro Design System.
+
+### The team's Product Designer submitted comps that contradict the specification of the Auro Design System.
+
+It is recommended that designers leverage Auro resources as much as possible. But there will be times when the design asset provided does not provide the assistance the designer needs to meed the goal of the new product work. 
+
+It is recommended that the discovered inconsistency is reviewed by the members of the Product Team to understand the reason for the inconsistency achieve a resolution. 
```

Comment:
**Missing conjunction**: "... for the inconsistency _and to_ achieve a resolution.

**Consideration**: Instead of using "achieve" perhaps it would be better to use "work towards a"